js/rpg_core/ImageCache.js

```diff
@@ -0,0 +1,124 @@
+function ImageCache(){
+    this.prototype.initialize.call(this);
```

Comment:
I grep source and I found It's mixed.

(this / ClassName).initialize.(apply/call)

Which shoud we use?